Jenkinsfile

```diff
@@ -1,6 +1,7 @@
 properties([
   buildDiscarder(logRotator(numToKeepStr: '50', artifactNumToKeepStr: '5')),
-  disableConcurrentBuilds(abortPrevious: true)
+  disableConcurrentBuilds(abortPrevious: true),
+  pipelineTriggers([cron('@weekly')]) // Run at least weekly to assure we test recent releases
```

Comment:
I don’t think we want this on non master branches

Reply:
Can you explain further why we would not want to run the test automation once a week on any branch that is active?

Once a branch becomes inactive (stable-2.319, stable-2.303, stable-2.289), we'd want to disable the weekly trigger of the job, but it seems safe and wise to assure that the test automation is working on both the weekly branch and the current stable branch.

I've submitted the revert of that change on the stable-2.332 branch as ea933557fa3d16d44a21e1153ccefc770126f592

Reply:
I think it's a waste of CI and we will forget to disable it. :)

Should mainline an on master only cron trigger I think.